packages/devtools_app/lib/src/debugger/flutter/debugger_screen.dart

```diff
@@ -26,55 +28,77 @@ class DebuggerScreen extends Screen {
-          textAlign: TextAlign.center,
-        ),
-      );
-    }
+  DebuggerScreenBodyState createState() => DebuggerScreenBodyState();
+}
 
-    // TODO(https://github.com/flutter/flutter/issues/43532): Don't build const
-    // because compile time const evaluation will fail on non-web apps.
-    // ignore:prefer_const_constructors
-    final webView = HtmlElementView(
-      viewType: 'DebuggerFlutterPlugin',
-    );
+class DebuggerScreenBodyState extends State<DebuggerScreenBody> {
+  DebuggerState debuggerState;
+  Script script;
 
-    // Wrap the content with an EagerGestureRecognizer to pass all mouse
-    // events to the web view.
-    return RawGestureDetector(
-      gestures: {
-        EagerGestureRecognizer: _EagerGestureFactory(PointerDeviceKind.mouse),
-      },
-      child: webView,
+  @override
+  void initState() {
+    super.initState();
+    debuggerState = DebuggerState();
+    debuggerState.setVmService(serviceManager.service);
+    // TODO(https://github.com/flutter/devtools/issues/1648): Make file picker.
+    serviceManager.service
+        .getScripts(serviceManager.isolateManager.selectedIsolate.id)
+        .then((scripts) async {
+      final scriptRef = scripts.scripts
+          .where((ref) => ref.uri.contains('package:flutter'))
+          .first;
+      print('script ref ${scriptRef.uri} found');
+      final _script = await serviceManager.service.getObject(
+        serviceManager.isolateManager.selectedIsolate.id,
+        scriptRef.id,
+      ) as Script;
+      print('${_script.library} loaded');
+      setState(() => script = _script);
+    });
+  }
+
+  @override
+  Widget build(BuildContext context) {
+    return Split(
+      axis: Axis.horizontal,
+      initialFirstFraction: 0.25,
+      // TODO(https://github.com/flutter/devtools/issues/1648): Debug panes.
+      firstChild: const Text('Debugger details'),
+      // TODO(https://github.com/flutter/devtools/issues/1648): Debug controls.
+      secondChild: CodeView(
+        script: script,
+      ),
     );
   }
 }
 
-class _EagerGestureFactory
-    extends GestureRecognizerFactory<EagerGestureRecognizer> {
-  _EagerGestureFactory(this.kind);
-
-  final PointerDeviceKind kind;
+class CodeView extends StatelessWidget {
+  const CodeView({Key key, this.script}) : super(key: key);
 
-  @override
-  EagerGestureRecognizer constructor() => EagerGestureRecognizer(kind: kind);
+  final Script script;
 
   @override
-  void initializer(EagerGestureRecognizer instance) {}
+  Widget build(BuildContext context) {
+    // TODO(https://github.com/flutter/devtools/issues/1648): Line numbers,
+    // syntax highlighting and breakpoint markers.
+    if (script == null) {
+      return const Center(child: CircularProgressIndicator());
+    }
+    return DefaultTextStyle(
+      style: Theme.of(context)
+          .textTheme
+          .bodyText2
+          .copyWith(fontFamily: 'RobotoMono'),
+      child: Scrollbar(
+        child: SingleChildScrollView(
```

Comment:
tiny nit: it would probably be clearer to have the 
`scrollDirection: Axis.vertical`
here even though it is the default.